packages/wonder-stuff-server-google/src/__tests__/get-app-engine-info.test.js

```diff
@@ -0,0 +1,63 @@
+// @flow
```

Comment:
Copied from render-gateway `getGatewayInfo` tests and modified to rename and update flow, etc.